app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  scope :three_most_recent_posts_for_user, ->(user_id) { where(id: user_id).order(created_at: :desc).limit(3) }
```

Comment:
Nice job so far😉. As a requirement, you are to create a METHOD that returns the three most recent posts, and not a scope as you have created😉. please do well to make the necessary changes to match the requirements👍🏼. also, I think the scope has an issue as it is filtering based on `id` column instead of the `user_id` column😉.

<img width="585" alt="image" src="https://github.com/ismayito/Rails-blog-app/assets/64643995/33e25991-17cf-4cae-a174-fe68c1e83c0e">


you can do this instead😉.

```
  def recent_post
    posts.order('created_at Desc').limit(3)
  end
```